### PR TITLE
[#IC-327] Added endpoint and pointed fn-app with upsert message status defined

### PR DESCRIPTION
**Pull request by michaeldisaro**
#### List of Changes

Added new endpoint to upsert message status attributes

This PR depends on pagopa/io-functions-app#244

#### Types of changes

- [ ] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by gquadrati — APPROVED** [review]

File: api_backend.yaml [hunk 1]
```diff
@@ -261,6 +261,54 @@ paths:
...
+          schema:
+            $ref: "#/definitions/MessageStatusChange"
+          required: true
+          x-examples:
+            application/json: |
+              change_type: "bulk",
+              is_archived: true,
+              is_read: true
+      responses:
+        "200":
```
Comment (balanza) [inline api_backend.yaml:287]: Nitpick: 

`204 No Content` is more appropriate as we're returning nothing (even `205 Reset` would make the case, although is a bit more exotic)

File: api_backend.yaml [hunk 2]
```diff
@@ -261,6 +261,54 @@ paths:
...
+        - name: body
+          in: body
+          schema:
+            $ref: "#/definitions/MessageStatusChange"
+          required: true
+          x-examples:
+            application/json: |
+              change_type: "bulk",
+              is_archived: true,
+              is_read: true
```
Comment (BurnedMarshal) [inline api_backend.yaml:285]: Can this property always set with any boolean value? I remember that a read message cannot be set to unread. Should be a specific status code for the forbidden operation (403 Forbidden / 409 Conflict).

Comment (BurnedMarshal) [reply api_backend.yaml:285]: Sorry, I see now that the type accepted for the is_read prop is only `true` value 😅

**Comment by pietro909** [comment]
Ping @balanza @BurnedMarshal - the PR was approved already. Is there anything preventing us from merging it?

**Comment by balanza** [comment]
@pietro909 Have you checked out this comment? https://github.com/pagopa/io-backend/pull/876/files#r823485999

**Comment by pietro909** [comment]
@balanza yes. It's unclear whether it's a blocker or not though. I don't have an opinion.

**Comment by balanza** [comment]
It's not a blocker. I assume you're fine with the current implementation so I merge.

Thanks!

**Pull request merged**